doc/concepts/cable_cell.rst

```diff
@@ -335,16 +335,24 @@ See :ref:`modelgapjunctions`.
 Discretisation and CV policies
 ------------------------------
 
-For the purpose of simulation, cable cells are decomposed into discrete
-subcomponents called *control volumes* (CVs). The CVs are
-uniquely determined by a set of *B* ``mlocation`` boundary points.
-For each non-terminal point *h* in *B*, there is a CV comprising the points
-{*x*: *h* ≤ *x* and ¬∃ *y* ∈ *B* s.t *h* < *y* < *x*}, where < and ≤ refer to the
-geometrical partial order of locations on the morphology. A fork point is
-owned by a CV if and only if all of its corresponding representative locations
-are in the CV.
+.. glossary::
 
-The set of boundary points used by the simulator is determined by a *CV policy*.
+  control volume
+    For the purpose of simulation, Arbor discretises cable cell :term:`morphologies <morphology>`
+    into control volumes, or CVs. Discretising happens through a :term:`CV policy`.
```

Comment:
That's almost certainly my fault. From my reading of the EU style guides, it probably should be 'discretise' and 'discretisation'. Though it almost hurts to type the latter :)

Reply:
I can set my spellchecker to en_GB or en_US, not both ;)

Reply:
I guess it's a question of consistency vs EU style compliance?

Reply:
If there's any doubt about which spelling to use, I think it best we follow the EU style guidelines, because then we'll be consistent across code, documentation, and reports. My personal failing in this is a tendency to use -ize endings — which is fine by the OED, but not the EU guidelines.

Reply:
I don't mind using `ise`, I'm just concerned about all our `discretize` functions not matching the documentation. It's a small issue though, so I don't mind leaving this as is.

Reply:
We can definitely leave it for now, but I'll make a ticket to remind me to revisit this later.

Reply:
Good point @noraabiakar, I prefer consistency over a particular spelling. This is also my beef with en_GB, nearly all software uses en_US spellings in APIs and documentation. Let's gather these points in the issue made by @halfflat.